Chat/Chat.cpp

```diff
@@ -0,0 +1,168 @@
+#include "Chat.h"
+#include "Network.h"
+#include <iostream>
+#include <thread>
+#include <chrono>
+
+#include <conio.h>
+
+using std::cout;
```

Comment:
You can left it here, but it is not good style . It is better to write std:: each time (despite the fact that code becomes bigger) Whatever...